src/modv_MXS.F

```diff
@@ -0,0 +1,16 @@
+C> @file
+C> @brief Declare and initialize MXS variable.
+
+C> This module declares and initializes the MXS variable.
+C>
+C> @author J. Ator
+C> @date 2021-03-30
+
+	MODULE MODV_MXS
+
+C>        @var mxs
+C>        Maximum number of entries in the internal string cache
+
```

Comment:
Please define private and public and only make mxs public.
This should be a `modv_MXS.f90`
There is no preprocessor involved here.

Reply:
OK, will do.  And there are a couple of other existing modf files which also don't have preprocessor directives, so I'll update those as well to keep things consistent (they're very small files), so that only modv files which really need to be preprocessed have the .F extension.

Going forward, I will also try to get in the habit of explicitly specifying PRIVATE vs. PUBLIC - good point!

Reply:
BTW, since MXS is already declared as a parameter and therefore (by definition) can't be changed by the outside code, then it would be more appropriate here to use `public, protected` here, rather than just `public`, correct?

Reply:
Never mind, it turns out `parameter` and `protected` are conflicting attributes (you can either have one or the other but not both).  So will just stick with `parameter` and `public` ;-)